.env.example

```diff
@@ -49,9 +49,7 @@ SSL_CERT_FILE_NAME=
 # set this env if you want to override the os.hostname() value
 HOSTNAME=NodeName
 
-PUSH_QUEUE_DRIVER=rsmq
+PUSH_QUEUE_DRIVER=sama_native
 
-SAMA_NATIVE_PUSH_QUEUE_NAME=TEST3
+SAMA_NATIVE_PUSH_QUEUE_NAME=sama_notification
```

Comment:
let's rename to `push_notifications` so it will be then clear in Bull Dashboard what the queue for